core/test/junit/saros/account/XMPPAccountStoreTest.java

```diff
@@ -89,26 +89,16 @@ private void writeAccountFile(File accountFile, String key, String content) thro
   }
 
   private String createAccountFileContent(
-      XMPPAccount activeAccount, List<XMPPAccount> otherAccounts) {
+      XMPPAccount activeAccount, ArrayList<XMPPAccount> configuredAccounts) {
 
+    Integer index = configuredAccounts.indexOf(activeAccount);
```

Comment:
It is "int" and not "integer" unless you want to return null. I have seen this multiple times in this patch set.

Reply:
Done